lib/ui/views/groups/group_details_view.dart

```diff
@@ -385,3 +396,38 @@ class _GroupDetailsViewState extends State<GroupDetailsView> {
     );
   }
 }
+
+class AddButton extends StatefulWidget {
+  AddButton({@required this.addFunction, this.context, @required Key key})
+      : super(key: key);
+  final Function addFunction;
+  final BuildContext context;
+  @override
+  _AddButtonState createState() => _AddButtonState();
+}
+
+class _AddButtonState extends State<AddButton> {
+  Function dynamicAddFunction;
+  void setAddFunction(bool isActive) {
+    if (isActive == true) {
+      setState(() {
+        dynamicAddFunction = widget.addFunction;
+      });
+    } else if (isActive == false) {
```

Comment:
same here, equals operator not required

Reply:
Ok, will refactor this in latest commit.